lib/process-services/src/lib/process-list/components/start-process.component.html

```diff
@@ -4,35 +4,50 @@
-            <mat-label>{{'ADF_PROCESS_LIST.START_PROCESS.FORM.LABEL.TYPE' | translate}}</mat-label>
-            <input
-                type="text"
-                matInput
-                [formControl]="processDefinitionInput"
-                [matAutocomplete]="auto"
-                id="processDefinitionName"
-                #inputAutocomplete>
-            <div class="adf-process-input-autocomplete">
-                <mat-autocomplete
-                    #auto="matAutocomplete"
-                    id="processDefinitionOptions"
-                    [displayWith]="displayFn">
-                    <mat-option *ngFor="let processDef of filteredProcesses | async" [value]="processDef.name"
-                    (click)="processDefinitionSelectionChanged(processDef)">
-                        {{ processDef.name }}
+        <div class="adf-start-process-definition-container">
+            <mat-form-field *ngIf="showSelectApplicationDropdown" [floatLabel]="'always'" class="adf-start-process-app-list">
+                <mat-select
+                    placeholder="{{'ADF_PROCESS_LIST.START_PROCESS.FORM.LABEL.APPLICATIONS' | translate}}"
+                    (selectionChange)="onAppSelectionChange($event)"
+                    [(ngModel)]="selectedApplication"
+                    data-automation-id="adf-start-process-apps-drop-down">
+                    <mat-option 
+                        *ngFor="let application of applications"
+                        [value]="application"
+                        [attr.data-automation-id]="'adf-start-process-apps-option' + application.name">
+                        {{ application.name }}
                     </mat-option>
-                </mat-autocomplete>
-                <button
-                    id="adf-select-process-dropdown"
-                    *ngIf="showSelectProcessDropdown"
-                    mat-icon-button
-                    (click)="displayDropdown($event)">
-                    <mat-icon>arrow_drop_down</mat-icon>
-                </button>
-            </div>
-        </mat-form-field>
-
+                </mat-select>
+            </mat-form-field>
+            <mat-form-field class="adf-process-input-container" [floatLabel]="'always'">
+                <mat-label>{{'ADF_PROCESS_LIST.START_PROCESS.FORM.LABEL.TYPE' | translate}}</mat-label>
+                <input
+                    type="text"
+                    matInput
+                    [formControl]="processDefinitionInput"
+                    [matAutocomplete]="auto"
+                    id="processDefinitionName"
+                    #inputAutocomplete>
+                <div class="adf-process-input-autocomplete">
+                    <mat-autocomplete
+                        #auto="matAutocomplete"
+                        id="processDefinitionOptions"
+                        [displayWith]="displayFn">
+                        <mat-option *ngFor="let processDef of filteredProcesses | async" [value]="processDef.name"
```

Comment:
`let processDef of filteredProcesses`. Can you rename this to `let processDef of filteredProcessDefinitions`? It should contain the definition keyword as it contains process definitions

Reply:
Updated